plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -770,6 +804,7 @@ private boolean createVMFullClone(VirtualMachineMO vmTemplate, DatacenterMO dcMo
         return true;
     }
 
+    //TODO: Remove this method after deploying VM from OVF directly using content library
```

Comment:
👍

Reply:
on second thought, we need to be backwards compatible. I don't think we can do this with a single transition script.